build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -190,6 +190,28 @@ def _convert_target_list_block(list_name, targets):
   return _convert_string_list_block(list_name, targets, sort=True, quote=False)
 
 
+# Copied from integrations/tensorflow/e2e/iree_e2e_cartesian_product_test_suite.bzl
+def _normalize_dictionary(dictionary):
+  """Wraps every value of dictionary in a list if it isn't one already."""
+  for key, value in dictionary.items():
+    if type(value) != type([]):
```

Comment:
`not isinstance(value, list)`

Reply:
These are directly copied from Starlark, so have some Starlark-oddities. Since it was copy-paste, I figured it wasn't worth updating unless they would actually cause an issue.